client/src/components/CustomTables.js

```diff
@@ -205,7 +228,13 @@ export function UpcomingOrOngoingTable(props) {
               <Typography>{interviewId}</Typography>
             </TableCell>
             <TableCell align="center">
-              <InsideTableButton text="Cancel" />
+              <Button
+                variant="outlined"
+                onClick={() => cancelInterview(interviewId)}
+              >
+                {" "}
+                Cancel{" "}
```

Comment:
Is this styled? You could reuse InsideTableButton component

Reply:
I tried using the InsideTableButton but the clickEvent is not working for some reason

Reply:
Please try this

//CustomButtons.js
export function InsideTableButton(props) {
  const classes = insideTableButtonStyles();

  function handleEvent(e) {
    e.preventDefault();
    props.clickEvent(e);
  }
  
  return (
    <Button variant="outlined" className={classes.button} onClick={handleEvent}>
      <Typography className={classes.text}>{props.text}</Typography>
    </Button>
  );
}

//CustomTables.js
<InsideTableButton text="Cancel" clickEvent={cancelInterview} />

Reply:
<InsideTableButton text="Cancel" clickEvent={cancelInterview} />

Reply:
`<InsideTableButton text="cancel" clickEvent={cancelInterview} />`